src/main/java/ejercicio/banco/dto/Account.java

```diff
@@ -2,7 +2,6 @@
 
 public class Account {
 
-    //Variables
     private int id;
     private String userName;
```

Comment:
Sorry if I didn't tell you before, but this is the same example as the `id`. We are in the `Account` class, so it makes no sense to say `userName` since I'm in the scope of the account already. 

In general, as a rule, we should never use the class name in a field. In this class, ideally we should see:

```
public class Account{
   private int id;
   private String name;
}
```